starter_code/app.js

```diff
@@ -1,21 +1,37 @@
 
 const express = require('express');
-const hbs     = require('hbs');
 const app     = express();
```

Comment:
Try to put all the requires at the top and separated from the res. You will get cleaner code.